### ISSUE TITLE
Hummus not rendering Thai diacritics properly

### ISSUE THREAD

**Issue by wickstargazer** [galkahana/HummusJS#189]
Hi,

We are facing a problem where the diacritic is not being rendered properly. Using the code from the example adding our own font and testing it with the language.

![screen shot 2017-08-17 at 5 01 39 pm](https://user-images.githubusercontent.com/1543442/29407200-d60cb5e2-836d-11e7-8bef-4520cff4bca9.png)

[CSChatThai.zip](https://github.com/galkahana/HummusJS/files/1230904/CSChatThai.zip)

The font..

**Comment by galkahana** [comment]
Right. diacritics are not supported by hummus, just plain unicode encoding.
There are two possible path:
1. if you are lucky to have specific unicodes for diacritics (In Arabic that's true) you can figure out yourself the cases where they should be used and provide them instead.
2. If you are less lucky, then you will have to figure out the font glyphs and send to hummus the glyph ids directly.

for Path number 2 you should be using the text drawing functions a little differently. see [here](https://github.com/galkahana/HummusJS/wiki/Use-the-pdf-drawing-operators#tj) for some explanation.

well there's also the option for you to add support for Thai diacritics in hummus directly, and everyone will thank you for that.
